docs/UserGuide.md

```diff
@@ -1215,7 +1213,8 @@ Important notes:
 
 1. `Amount` entered has to be positive for any `interval` value.
 2. `Amount` is automatically rounded to 2 decimal places.
-3. `Interval` must be of values "day", "week", or "month".
+3. `Amount` must be less than `2,147,483,647`.
```

Comment:
So just to confirm, are we sticking to this limit, or do we want to consider limiting the max amount that the user can input as well, perhaps to $100,000? No problems if we stick to this max limit, but in the case that we are asked to justify we can state that our limitation is more appropriate for our target user.

No technical issues though, just a product design/marketing concern.

Reply:
For now this is the limit we've set for ourselves, I think we only need to change it if there's a bug report on it, also, think @lipwei1808 would be the one deciding since he made Amount

Reply:
Yup sounds good, no worries there, just something to consider holistically in general as a team since there's rippling effects across other components, but for now no issues at all since everything works fine